src/pe/mod.rs

```diff
@@ -273,14 +273,15 @@ impl<'a> PE<'a> {
 
             if let Some(&load_config_dir) = optional_header.data_directories.get_load_config_table()
             {
-                load_config_data = Some(load_config::LoadConfigData::parse_with_opts(
+                load_config_data = load_config::LoadConfigData::parse_with_opts(
                     bytes,
                     load_config_dir,
                     &sections,
                     file_alignment,
                     opts,
                     is_64,
-                )?);
+                )
+                .ok();
```

Comment:
Are you sure we want to drop all errors when parsing and return none if malformed ? It’s ok to gracefully error when fuzzing (eg your fix is good) but for regular uses if the config is malformed should we not continue parsing ?

Or perhaps we should check the strict flag to stop otherwise return none, seems like a good compromise ?

Reply:
It's good if we could defer the parse in some way so that the user could parse the rest of binary. This change is not mandatory so I am going to revert this part atm